ASTProcessor/src/main/java/edu/usb/argos/ASTProcessor/JavaParserExample.java

```diff
@@ -0,0 +1,31 @@
+package edu.usb.argos.ASTProcessor;
+
+import com.github.javaparser.JavaParser;
+import com.github.javaparser.ast.CompilationUnit;
+import com.github.javaparser.ast.body.MethodDeclaration;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class JavaParserExample {
```

Comment:
When we have referred to the file to test you could have added one more class implementation based on the diagram defined on lucidChart

Reply:
I didn't understand very well, could you give me a little more information or could you explain it to me in another way please?

Reply:
According to what was discussed in class, the tests were eliminated since the pull requests will already comply with the tests that are needed and this will only have the initialization of the javaparser library.